src/infra/security/jwt.token-manager.ts

```diff
@@ -0,0 +1,25 @@
+import jwt from "jsonwebtoken";
+import { TokenManager } from "../../domain/ports/token-manager";
+
+export class JwtTokenManager implements TokenManager {
+  constructor(
+    private readonly accessSecret: string,
+    private readonly refreshSecret: string
+  ) {}
+
+  signAccessToken(payload: { id: string; nickname: string }): string {
+    return jwt.sign(payload, this.accessSecret, { expiresIn: "15m" });
+  }
+
+  signRefreshToken(payload: { id: string; nickname: string }): string {
+    return jwt.sign(payload, this.refreshSecret, { expiresIn: "7d" });
+  }
+
+  verifyAccessToken(token: string) {
+    return jwt.verify(token, this.accessSecret) as any;
```

Comment:
any보다는

```suggestion
verifyAccessToken(token: string): { id: string; nickname: string } {
  try {
    const payload = jwt.verify(token, this.accessSecret);
    return payload as { id: string; nickname: string };
  } catch (error) {
    throw new UnauthorizedException('Invalid access token');
  }
}
```

같이 type 추가 및 에러 핸들링을 해봐도 좋을꺼 같아요~